locales/it/index.yml

```
@@ -1032,6 +1032,12 @@ identification:
       title: Identificazione richiesta
       sensorDescription: Tocca il sensore
       fallbackLabel: Usa il codice di sblocco
+  fail:
+    wrongCode: Codice errato
+    remainingAttempts: Ti rimangono {{attempts}} tentativi.
+    remainingAttemptSingle: Ti rimane {{attempts}} tentativo.
+    tooManyAttempts: Troppi tentativi di inserimento errati.
+    waitMessage: "Chiudi l'applicazione e riprova tra:"
```

```suggestion
    waitMessage: "Riprova tra:"
```
Since we don't offer any way to close the app, I find this message more appropriate.
What do you think?